src/components/dialogs/resubmit.vue

```diff
@@ -311,7 +311,7 @@ export default class ResubmitDialog extends Mixins(
 </script>
 
 <style lang="scss" scoped>
-@import "@/assets/scss/theme";
+@import "@/assets/styles/theme";
```

Comment:
I just noticed this now!

This is an invalid import. It didn't show as an error because none of the SCSS statements here use any variabled from theme.scss.

Please fix this: either fix or delete this import (since it's not doing anything). Check the other files, too.

Reply:
Even tested this (in request-details.vue) and it seems this import works after all. (I just learned something.)

Therefore this is not an error. No need to change the imports.